src/provider/view/components/Header/Header.tsx

```diff
@@ -65,12 +67,26 @@ interface LockProps extends Actionable {
     centerLocked: boolean;
 }
 
+interface MuteProps extends Actionable {
+    centerMuted: boolean;
+}
+
 function Lock(props: LockProps): React.ReactElement {
     const {storeApi, centerLocked} = props;
 
-    const handleLockWindow = () => {
-        new ToggleLockCenter().dispatch(storeApi);
+    const handleLockCenter = () => {
+        new ToggleCenterLocked().dispatch(storeApi);
+    };
+
+    return <a id="lock-link" className="developmentOnly" onClick={handleLockCenter}>{centerLocked ? '🔒' : '🔓'}</a>;
+}
+
+function Mute(props: MuteProps): React.ReactElement {
+    const {storeApi, centerMuted} = props;
+
+    const handleMuteCenter = () => {
+        new ToggleCenterMuted().dispatch(storeApi);
     };
 
-    return <a id="lock-link" onClick={handleLockWindow}>{centerLocked ? '🔒' : '🔓'}</a>;
+    return <a id="mute-link" className="developmentOnly" onClick={handleMuteCenter}>{centerMuted ? '🔈' : '🔊'}</a>;
```

Comment:
Very minor detail that doesn't really matter - the clickbox for these two images are different, so if you click the soundwaves, you can't click in the same spot to unmute it.

Reply:
If you add some to the `mute-link` it will fix that.

Reply:
@TheoOttahOF Not sure I understand...

Reply:
(Though also this goes away by itself once the settings UI is in)